include/mxnet/ndarray.h

```diff
@@ -305,7 +305,10 @@ class NDArray {
   bool fresh_out_grad() const;
   /*! \return updated grad state in entry_ */
   void set_fresh_out_grad(bool state) const;
-  // returns true if a sparse ndarray's aux_data and storage are initialized
+  /*! \brief Returns true if a sparse ndarray's aux_data and storage are initialized
+   * Returns false if the indices array shape is inconsistent
```

Comment:
"Returns false if the indices array shape is inconsistent" -> it actually throws an exception without returning false